cg/models/scout/scout_load_config.py

```diff
@@ -145,3 +150,10 @@ class RnafusionLoadConfig(ScoutLoadConfig):
     RNAfusion_report_research: str | None = None
     samples: list[ScoutCancerIndividual] = []
     vcf_fusion: str | None = None
+
+
+class TomteLoadConfig(ScoutLoadConfig):
+    multiqc_rna: str | None = None
+    samples: list[ScoutRnaIndividual] = []
+    vcf_snv: Annotated[str, BeforeValidator(field_not_none)] = None
+    vcf_snv_research: Annotated[str | None, BeforeValidator(field_not_none)] = None
```

Comment:
All of these attributes are duplicated and we could just inherit them from a RareDisease and RNA classes:

`TomteLoadConfig(RareDiseaseLoadConfig, RNALoadConfig)`

Reply:
Could also fit into a future refactoring, which is much needed for this API 🥵

Reply:
I would leave this for now. It should be done in a different pr